pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -468,3 +490,19 @@ func getPVVolumeGidAnnotationValue(pv *v1.PersistentVolume) string {
 
 	return ""
 }
+
+func makeVolumeDevicesMap(containers []v1.Container) map[string]bool {
+	volumeDevicesMap := make(map[string]bool)
+
+	for _, container := range containers {
+		if container.VolumeDevices == nil {
+			continue
+		}
+		for _, device := range container.VolumeDevices {
+			glog.Infof("#### DEBUG LOG ####: makeVolumeDevicesMap: device: %v", device)
+			volumeDevicesMap[device.Name] = true
```

Comment:
Is it possible to have the same device name in two different containers, but referencing two different volumes?